mvpa2/algorithms/hyperalignment.py

```diff
@@ -198,6 +198,10 @@ def train(self, datasets):
         """
         params = self.params            # for quicker access ;)
         ca = self.ca
+        # Check to make sure we get a list of datasets as input.
+        if not (isinstance(datasets, list) or isinstance(datasets, tuple)):
```

Comment:
1. could be just `isinstance(datasets, (list, tuple, np.ndarray))` I believe
2. doc says it should be a "sequence", so could be something else than a list or a tuple I guess (e.g. a ndarray) -- something which we could iterate over and which has an order to it (so shouldn't be a set)... list/tuple/ndarray of them indeed. please adjust docstring and use above construct instead of `or`s